lib/utils.js

```diff
@@ -620,8 +626,8 @@ export function synchronizeLayoutWithChildren(
   });
 
   // Correct the layout.
-  layout = correctBounds(layout, { cols: cols });
-  layout = compact(layout, compactType, cols);
+  // layout = correctBounds(layout, { cols: cols });
```

Comment:
In which cases not calling these methods can bite us ? Only in not controlled mode ?